tests/unit/test_base.py

```diff
@@ -50,6 +50,10 @@ def test_test_invalid():
     # Invalid attributes
     for key in ['component', 'require', 'tag']:
         with pytest.raises(tmt.utils.SpecificationError):
-            test = tmt.Test({key: 'string'}, name='/smoke')
+            test = tmt.Test({key: 1}, name='/smoke')
     with pytest.raises(tmt.utils.SpecificationError):
         test = tmt.Test({'environment': 'string'}, name='/smoke')
+    # Listify attributes
+    assert tmt.Test({'tag': 'a'}, name='/smoke').tag == ['a']
+    assert tmt.Test({'tag': 'a b c'}, name='/smoke').tag == ['a b c']
```

Comment:
I understand this is test of listify, however per documentation input value is invalid (comma, space separation). Maybe to pick another attribute (where spaces and commas are allowed)?

Reply:
Right your are, now it does not make sense here at all. Removing it as `listify` has its own coverage.